src/Equinor.ProCoSys.IPO.Query/GetInvitationsQueries/GetInvitationsForExport/GetInvitationsForExcelQueryHandler.cs

```diff
@@ -44,10 +44,15 @@ public async Task<Result<ExportDto>> Handle(GetInvitationsForExportQuery request
             var invitationIds = orderedDtos.Select(dto => dto.Id).ToList();
             var getHistoryAndParticipants = invitationIds.Count == 1;
 
-            var invitationsWithIncludes =
-                await GetInvitationsWithIncludesAsync(invitationIds, getHistoryAndParticipants, cancellationToken);
+            var invitationsWithIncludes = await (from invitation in _context.QuerySet<Invitation>()
```

Comment:
make this block as protected method GetInvitationsWithIncludesAsync() in base class and reuse on both "child" classes